### PR TITLE
Added looping cascade to the popouts

### PR DESCRIPTION
**Pull request by ddbrown30**

### CODE REVIEW

File: betterrolls-swade2/scripts/cards_common.js [hunk 1]
```diff
@@ -129,7 +131,10 @@ export class BrCommonCard {
       return;
     }
     if (game.settings.get("betterrolls-swade2", "auto_popout_chat")) {
-      new ChatPopout(this.message).render(true);
+      let top = 250 + (cascade_count * 35);
+      let left = 700 + (cascade_count * 20);
+      cascade_count = cascade_count < 3 ? cascade_count + 1 : 0;
```
[the remark this answers is not in the document]
Comment (ddbrown30) [reply betterrolls-swade2/scripts/cards_common.js]: Up to you if you want to do this. Foundry has a minimum resolution so I'm not too fussed about trying to support different resolutions.

[the remark this answers is not in the document]
Comment (ddbrown30) [reply betterrolls-swade2/scripts/cards_common.js]: I went a step further and converted everything here to config variables. My only question is if you prefer lower case or upper case for vars like this.

[the remark this answers is not in the document]
Comment (javierriveracastro) [reply betterrolls-swade2/scripts/cards_common.js]: I agree, all Foundry is pixel coded (and that is a problem to some people), so it doesn't make sense to complicate the code for this. Anyway, the bot is getting really smart.

[the remark this answers is not in the document]
Comment (javierriveracastro) [reply betterrolls-swade2/scripts/cards_common.js]: Currently the code uses upper case for "configuration constants" and lower case for config options.

**Pull request merged**